frontend/components/admin/approve-users/approve-users.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Container, List, Spinner} from "@chakra-ui/react";
+import { Fragment, useState } from "react";
+import UserList from "@frontend/components/admin/list";
+import UserInfoSimple from "@frontend/models/user-info-simple";
+import unconfirmedUsersList from "@frontend/hooks/unconfirmed-users-list";
+import ApproveUsersRowCard from "@frontend/components/admin/approve-users/approve-users-row-card";
+
+interface appProps {
+    sessionId: Number;
+}
+
+export function getPendingUsers(){
+    //let { users, isLoading, isError } = unconfirmedUsersList(); // Custom Hook to Fetch the user/role data
+    const data = unconfirmedUsersList();
+    return data;
+}
+
+export function ApproveUsers (users: any, isLoading: boolean, isError: any, sessionId: Number)  {
+    //let { users, isLoading, isError } = unconfirmedUsersList(); // Custom Hook to Fetch the user/role data
+    //console.log(users);
+
+    const [userSelected, setUserSelected] = useState<UserInfoSimple>({
+        AccountId: 0,
```

Comment:
this could maybe be moved to a dummy file for default data

Reply:
this is actually dead code, thanks for bringing it to my attention lmao